.github/dependabot.yml

```diff
@@ -47,6 +49,7 @@ updates:
       # Kafka
       - dependency-name: org.apache.kafka:*
       - dependency-name: org.apache.zookeeper:zookeeper
+      - dependency-name: org.xerial.snappy:snappy-java
```

Comment:
@cescoffier not sure about this one?

Reply:
This is Kafka related, it should come from Kafka, however, because we have funky substitutions, we need to control the version.
@ozangunalp WDYT?

Reply:
That doesn't mean that you have to accept all the updates. It's more like: do you want to be notified of updates.

Reply:
This one is pretty much pegged to what kafka-client is pulling. I think we won't be changing its version independently.

Reply:
so, let's remove it.

Reply:
Done.